tests/integration/test_integration_target_postgres.py

```diff
@@ -11,9 +11,9 @@
 from psycopg2.errors import InvalidTextRepresentation
 
 try:
-    import tests.utils as test_utils
+    import tests.utils as test_utils  # type: ignore
```

Comment:
Why do we have changes in the `tests` folder?

Reply:
Fixed